### PR TITLE
Add link to go CLI with tldr-pages pre-packaged

### PR DESCRIPTION
**Pull request by leighmcculloch**
Add link to go CLI with tldr-pages pre-packaged

----

~~- [ ] The page (if new), does not already exist in the repo.~~

~~- [ ] The page (if new), has been added to the correct platform folder:  ~~
~~      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.~~

~~- [ ] The page has 8 or fewer examples.~~

~~- [ ] The PR is appropriately titled:  ~~
~~      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited~~

~~- [ ] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines~~

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,6 +68,7 @@ You can access these pages on your computer using one of the following clients:
 - [Elixir client](https://github.com/edgurgel/tldr_elixir_client) (binaries not yet available)
 - [Emacs client](https://github.com/kuanyui/tldr.el), available on [MELPA](https://github.com/melpa/melpa)
 - [Go client](https://github.com/pranavraja/tldr): `go get github.com/pranavraja/tldr` (or [platform binaries](https://github.com/pranavraja/tldr/releases))
+- [Go pre-packaged](https://github.com/leighmcculloch/tldr): `go get 4d63.com/tldr` or `brew install leighmcculloch/tldr/tldr` or [binaries](https://github.com/leighmcculloch/tldr/releases)
```
Comment (agnivade) [inline README.md]: Lets keep that main link as [Go client] for consistency.

You can split the description into 2 lines. First line can be the `go get` one. And second can have the `brew install` and binaries link.

Comment (leighmcculloch) [reply README.md]: I made the change you requested, but then realized the other languages that have multiple all group under a single one for each language. I just pushed another change to bring it inline with that, do you prefer the latest?

**Comment by agnivade** [comment]
This is great. 

Slight nitpick on the platform binaries thing. The one above has parentheses, yours doesn't. I don't have any preference. But let's make both consistent.

**Comment by agnivade** [comment]
Please sign the CLA and then after a 2nd approval, we can get to merging this.

**Comment by leighmcculloch** [comment]
@agnivade @sbrl: Thanks! I'll sign the CLA shortly. As a courtesy any CLAs I sign are 👀  by my employer and it can take a few days for them to respond, but I'll sign it as soon as I get a response.

**Comment by agnivade** [comment]
Interesting. So does that mean your employer owns your open source work ? I would think whatever you do in your free time if your own. :wink:

**Comment by leighmcculloch** [comment]
They don't in most situations, it's a courtesy.

**Comment by leighmcculloch** [comment]
@agnivade @sbrl CLA signed.

**Pull request merged**